flaps/flaps_machines.go

```diff
@@ -1,10 +1,13 @@
+// FIXME: how do we handle flaps calls timing out. We still need to send metrics for it
```

Comment:
Kind of out of scope for this PR, plus it's a miniscule number of metrics that we don't need to worry about for right now.